assets/js/toggle_view.js

```diff
@@ -87,3 +89,7 @@ if (oidcUserInfoTab1Button) {
   oidcUserInfoTab1Button.addEventListener('click', showTabOneCode);
   oidcUserInfoTab2Button.addEventListener('click', showTabTwoCode);
 }
+if (oidcLogoutTab1Button) {
+  oidcLogoutTab1Button.addEventListener('click', showTabOneCode);
+  oidcLogoutTab2Button.addEventListener('click', showTabTwoCode);
+}
```

Comment:
Is there any way we could generalize this so we don't need to have 4 different sets of conditions? e.g. if we applied a common class to the tab buttons so that we add an event listener to whichever element has that class, regardless which one it is?